### ISSUE TITLE
undefined reference to `nvglImageHandle` in nanovg_gl_utils.h

### ISSUE THREAD

**Issue by IceDragon200** [memononen/nanovg#263]
```
nanovg_gl_utils.h:67: undefined reference to `nvglImageHandle'
```

This is from the HEAD while trying to build the examples
All I did was pull the latest master

Steps to reproduce (?)
```bash
git checkout master
git pull origin master
# I had an old build folder
rm -rf build 
premake4 gmake
cd build && make
```

**Comment by memononen** [comment]
Sorry about that, the PR #201 seemed to broke this, fixed. Thanks for reporting it! Fixed in b83cf926525e7cea8d2483da2a75852b8c7b6d28

**Comment by IceDragon200** [comment]
@memononen Awesome, that was fast thanks!
